impeller/renderer/backend/gles/capabilities_gles.cc

```diff
@@ -106,6 +106,21 @@ CapabilitiesGLES::CapabilitiesGLES(const ProcTableGLES& gl) {
       gl.GetDescription()->HasExtension(kOESTextureBorderClampExt)) {
     supports_decal_sampler_address_mode_ = true;
   }
+
+  if (gl.GetDescription()->HasExtension(
```

Comment:
Since we depend on the blit framebuffer proc right now, we should also check if its supported here and refuse MSAA otherwise.

Reply:
Is that not what I did below 😅

Reply:
oh thats good I'm just dumb